src/core/Akka.Streams.Tests/IO/TcpSpec.cs

```diff
@@ -457,14 +458,14 @@ public void Outgoing_TCP_stream_must_handle_when_connection_actor_terminates_une
                 .RunAggregate(0, (i, s) => i + s.Count, mat2);
 
             // Getting rid of existing connection actors by using a blunt instrument
-            system2.ActorSelection(system2.Tcp().Path/"selectors"/"$b"/"*").Tell(Kill.Instance);
+            system2.ActorSelection(system2.Tcp().Path/"$a"/"*").Tell(Kill.Instance);
```

Comment:
This line made me do a triple take :P Never new the ActorPath class implements the / operator